### DISCUSSION TITLE
GitOps Version Control Events

### DISCUSSION THREAD

**Discussion by TracyRagan** [cdfoundation/sig-events#30] [General]
We need to discuss and clarify events in Version Control to include what may be needed to automate the updating of .yaml files in a GitOps model. What GitOps events should be included, and should this be a separate POC?

**Comment by afrittoli** [comment]
GitOps are a kind of deployment events, which can have different granularity
- new configuration
- new image version
- new traits (runtime aspects, like replicas)
- configuration drift event
- deployment complete event

It would be interesting to get this events outside of the gitops operator.
Flux calls these [notifications](https://toolkit.fluxcd.io/guides/notifications/)

**Comment by afrittoli** [comment]
We could reach out to the CNCF GitOps WG: https://github.com/gitops-working-group/gitops-working-group-old
